app/helpers/checklist_tests_helper.rb

```diff
@@ -59,6 +59,19 @@ def coded_attribute?(criteria)
   def lookup_valueset_name(oid)
     vs = HealthDataStandards::SVS::ValueSet.where(oid: oid)
     return oid unless vs && vs.first
-    "#{vs.first.display_name}: #{oid}"
+    "#{vs.first.display_name}"
+  end
+
+
+  def lookup_valueset_oid(oid)
+    vs = HealthDataStandards::SVS::ValueSet.where(oid: oid)
+    return oid unless vs && vs.first
+    "#{oid}"
```

Comment:
I think you probably want this function to do something different, but I'm not sure. Right now it will just return `oid` every time